### PR TITLE
Pin maven-shade-plugin version

### PR DESCRIPTION
**Pull request by kokosing**
Pin maven-shade-plugin version

### CODE REVIEW

File: airbase/pom.xml [hunk 1]
```diff
@@ -879,6 +879,12 @@
                     <version>3.2</version>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-shade-plugin</artifactId>
+                    <version>3.1.1</version>
```
Comment (electrum) [inline airbase/pom.xml]: The latest is `3.2.1`, so we should use that.

File: airbase/pom.xml [hunk 2]
```diff
@@ -879,6 +879,12 @@
...
+                </plugin>
+
```
Comment (electrum) [inline airbase/pom.xml]: Nit: no need for a blank line between closing tags

**Comment by kokosing** [comment]
Updated

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**